src/EditorFeatures/CSharpTest/InlineMethod/CSharpInlineMethodTests.cs

```diff
@@ -0,0 +1,645 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.CodeRefactorings.InlineMethod;
+using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.CodeRefactorings;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.InlineMethod
+{
+    [Trait(Traits.Feature, Traits.Features.CodeActionsInlineMethod)]
+    public class CSharpInlineMethodTests : AbstractCSharpCodeActionTest
+    {
+        protected override CodeRefactoringProvider CreateCodeRefactoringProvider(Workspace workspace, TestParameters parameters)
+            => ((TestWorkspace)workspace).ExportProvider.GetExportedValue<CSharpInlineMethodRefactoringProvider>();
```

Comment:
i'd recommend just using the new test-harness system.

Reply:
Could you give a link to an example of the new test system?

Reply:
yup!  take a look at the GenerateEqualsAndGetHashCodeFromMembersTests